src/backend/models/club.js

```diff
@@ -31,5 +33,18 @@ Club.belongsTo(Base_profile, { foreignKey: "id" });
 Club.belongsTo(Council, { foreignKey: "councilId" });
 Council.hasMany(Club);
 
+Hackathon.hasOne(Club, {
+	foreignKey: "id",
+});
+Club.belongsTo(Hackathon, {
+	foreignKey: "club_id",
+});
+
+Project.hasOne(Club, {
+	foreignKey: "id",
```

Comment:
Why are you defining a one-to-one relationship here, when Project to Club is many-to-one relationship. And you have already defined that many-to-one relationship in `models/project.js`